client/me/billing-history/receipt.jsx

```diff
@@ -3,51 +3,56 @@
 /**
  * External dependencies
  */
-
 import React from 'react';
-import createReactClass from 'create-react-class';
+import page from 'page';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import page from 'page';
 
 /**
  * Internal dependencies
  */
-import tableRows from './table-rows';
-import eventRecorder from 'me/event-recorder';
+import Button from 'components/button';
 import Card from 'components/card';
-import Main from 'components/main';
-import HeaderCake from 'components/header-cake';
 import DocumentHead from 'components/data/document-head';
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
-import QueryBillingTransactions from 'components/data/query-billing-transactions';
 import purchasesPaths from 'me/purchases/paths';
+import QueryBillingTransactions from 'components/data/query-billing-transactions';
+import tableRows from './table-rows';
 import { getPastBillingTransaction, getPastBillingTransactions } from 'state/selectors';
+import { recordGoogleEvent } from 'state/analytics/actions';
 
-const BillingReceipt = createReactClass( {
-	displayName: 'BillingReceipt',
-	mixins: [ eventRecorder ],
-
+class BillingReceipt extends React.Component {
 	componentDidMount() {
 		this.redirectIfInvalidTransaction();
-	},
+	}
 
 	componentDidUpdate() {
 		this.redirectIfInvalidTransaction();
-	},
+	}
+
+	recordClickEvent = action => {
+		this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
+	};
+
+	handleSupportLinkClick = () => {
+		this.recordClickEvent( 'Contact {appName} Support in Billing History Receipt' );
+	};
+
+	handlePrintLinkClick = event => {
+		event.preventDefault();
```

Comment:
Do we need the `preventDefault`?

Reply:
Actually, we don't. Removed in c9c821f19c7b28d9047fba132e9c1148e4a1c879